packages/scm/src/browser/scm-tree-widget.tsx

```diff
@@ -567,7 +588,7 @@ export class ScmResourceComponent extends ScmElement<ScmResourceComponent.Props>
                 model,
                 treeNode
             }}>
-                <div title={tooltip} className='status' style={{ color }}>
+                <div title={tooltip} className={[codiconClass, classType].filter(Boolean).join(' ')} style={{ color, backgroundImage }} >
```

Comment:
Why is the `.filter` needed?

Reply:
It otherwise leaves a trailing space in front of the class name when there are no codicon classes. Might eventually not be necessary.